src/components/Button.jsx

```diff
@@ -167,6 +167,14 @@ const Button = ({
   if (small) className.push('small')
   className = className.join(' ')
 
+  const longArrowOrTextArrow = () => {
```

Comment:
Let's give this a shorter name. I'd call this function something like `handleArrow`. That way if someone were looking at just the markup they'll immediately know what that function does without looking at the code in the function.

Reply:
Ok. Thanks for this. I thought it may be too long. I juggled around `renderArrow` and `displayArrow`